draft-ietf-tls-esni.md

```diff
@@ -1340,18 +1362,20 @@ ClientHello vulnerable to an analogue of this attack.
 
 ## Update of the TLS ExtensionType Registry
 
-IANA is requested to create the following two entries in the existing registry
+IANA is requested to create the following three entries in the existing registry
 for ExtensionType (defined in {{!RFC8446}}):
 
-1. encrypted_client_hello(0xfe08), with "TLS 1.3" column values being set to
-   "CH, EE", and "Recommended" column being set to "Yes".
-3. outer_extensions(0xfd00), with the "TLS 1.3" column values being set to "",
-   and "Recommended" column being set to "Yes".
+1. encrypted_client_hello(0xfe09), with "TLS 1.3" column values set to
+   "CH, EE", and "Recommended" column set to "Yes".
+2. is_client_hello_inner (0xda09), with "TLS 1.3" column values set to
+   "CH", and "Recommended" column set to "Yes".
+3. outer_extensions(0xfd00), with the "TLS 1.3" column values set to "",
+   and "Recommended" column set to "Yes".
```

Comment:
My bike shed proposal is to use the following names:
1. "encrypted_client_hello"
2. "ech_is_inner"
3. "ech_outer_extensions"

That way the extensions that are only meaningful to ECH have names that indicates as much.

Reply:
Ack, dangit, I forgot to change "outer_extensions." Sorry about that!

Reply:
No biggy!